practice-one/README.md

```diff
@@ -0,0 +1,80 @@
+* [JSON](https://www.w3schools.com/js/js_json_intro.asp)
+  + JSON stands for JavaScript Object Notation, which is a data format that follows a certain rule that most programming languages today can read. JSON is an open standard for data interchange on the web
+* [React](https://react.dev/learn)
+  + React is a JavaScript library for building reusable user interfaces efficiently. It uses a virtual DOM for optimal rendering and updates only what's necessary. With a component-based approach, React simplifies application development and maintenance. It's popular for its simplicity, flexibility, and strong community support.
+
+## DEVELOPMENT TOOLS:
+
+* [Prettier](https://docs.google.com/document/d/1MF8DaLU2YsnOjFPMEMo7l8BASeBk6MHj2dXO-IhWXag/edit#heading=h.wn1dmnyf0s8c)
+    - Prettier is a code formatting tool that automatically formats code based on predefined rules. It supports various programming languages, integrates with popular code editors, and promotes consistent code style. It reduces debates, enhances code readability, and saves time on manual formatting tasks.
+* [Eslint](https://eslint.org/)
+    - ESLint is a popular JavaScript code analysis tool. It identifies and fixes issues, enforces coding conventions, and improves code quality. By applying configurable rules, it catches errors, enforces best practices, and provides real-time feedback. ESLint integrates easily and enhances code quality, readability, and maintainability in JavaScript projects.
+* [Vite](https://vitejs.dev/)
+    - Vite is a lightweight build tool for modern web apps. It supports fast development with instant updates, native ES module handling, and efficient bundling for production. It improves performance and enables quick iteration in JavaScript applications.
+
+## TARGET
+
+* Apply what you have read about the main concepts of ReactJS
+* Can apply ReactJS to build a  website
+* Understand and build the common components
+* Apply what you have read about the main concepts of ReactJS to rewrite your previous HTML/CSS practice into React components.
+
+## HOW TO START
+
+* Step 1 Clone repository:
+    - With HTTPS:
+    ~~~
+    git clone https://github.com/Agility-Internship/Duong-The-Hien-ReactJS.git
+    ~~~
+
+* Step 2 Move to folder which just cloned in your computer:
+    ~~~
+    cd Duong-The-Hien-ReactJS
+    ~~~
+
+* Step 3 Checkout into practice-02 branch:
+    ~~~
+    git checkout practice-one
+    ~~~
+
+* Step 4 Move to practice-02:
+    ~~~
+    cd practice-one
+    ~~~
+
+* Step 5 Now you need to install packages:
+    ~~~
+    pnpm install
+    ~~~
+
+* Step 6 After installing the packages:
+    ~~~
+    pnpm start
+    ~~~
+
+* Step 7 Open **http://localhost:5173/src/** in browser

```

Comment:
Check this link

Reply:
Checked and it is correct link when running pnpm dev command